app/main.py

```diff
@@ -24,20 +24,20 @@ class Cat(Animal):
     def __init__(self, name: str, is_hungry: bool = True) -> None:
         super().__init__(name, 3, is_hungry)
 
-    def catch_mouse(self) -> None:
+    @staticmethod
+    def catch_mouse() -> None:
         print("The hunt began!")
 
 
 class Dog(Animal):
     def __init__(self, name: str, is_hungry: bool = True) -> None:
         super().__init__(name, 7, is_hungry)
 
-    def bring_slippers(self) -> None:
+    @staticmethod
+    def bring_slippers() -> None:
         print("The slippers delivered!")
 
 
 def feed_animals(list_of_animals: list[Animal]) -> int:
-    food_points = 0
-    for animal in list_of_animals:
-        food_points += int(animal.feed())
-    return food_points
+    food_points_list = [int(animal.feed()) for animal in list_of_animals]
+    return sum(food_points_list)
```

Comment:
you can do it to use only sum: `return sum(int(animal.feed()) for animal in list_of_animals)
 `